### PR TITLE
feat(config): add HomeSeer HS-WX300 v1.13 parameters

### PR DESCRIPTION
**Pull request by McLain114**
1.13 firmware was recently released for HS-WX300 which includes some features that would be beneficial to have exposed to ZWave-JS. This is my attempt at adding this config.

Please let me know if I'm going about adding these incorrectly or there are any suggested changes.

### CODE REVIEW

File: packages/config/config/devices/0x000c/hs-wx300.json [hunk 1]
```diff
@@ -161,6 +161,81 @@
...
+			"options": [
+				{
+					"label": "LEDs do not flash on startup",
+					"value": 0
+				},
+				{
+					"label": "LEDs flash on startup to indicate switch or dimmer operation",
+					"value": 1
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x000c/hs-wx300.json]:
```suggestion
			"$import": "~/templates/master_template.json#base_enable_disable",
			"label": "Indicate Operation Mode On Startup"
```
(I hope this is what the parameter is supposed to do, I wasn't quite sure)

File: packages/config/config/devices/0x000c/hs-wx300.json [hunk 2]
```diff
@@ -161,6 +161,81 @@
...
+			"unsigned": true
+		},
+		{
+			"#": "35",
+			"label": "Paddle Behavior for Switch Mode (Toggle Feature)",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x000c/hs-wx300.json]:
```suggestion
			"label": "Paddle Behavior (Switch Mode)",
			"valueSize": 1,
			"defaultValue": 0,
			"allowManualEntry": false,
```

Comment (AlCalzone) [reply packages/config/config/devices/0x000c/hs-wx300.json]: min/max value are inferred from the options, but allowManualEntry must be false to disallow any other values.

File: packages/config/config/devices/0x000c/hs-wx300.json [hunk 3]
```diff
@@ -161,6 +161,81 @@
...
+			"options": [
+				{
+					"label": "Top paddle press turns load ON. Bottom paddle press turns load OFF.",
+					"value": 0
+				},
+				{
+					"label": "Either paddle press turns load ON if OFF and OFF if ON.",
+					"value": 1
+				}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x000c/hs-wx300.json]:
```suggestion
				{
					"label": "Top paddle turns on, Bottom paddle turns off",
					"value": 0
				},
				{
					"label": "Both paddles toggle",
					"value": 1
				}
```

File: packages/config/config/devices/0x000c/hs-wx300.json [hunk 4]
```diff
@@ -161,6 +161,81 @@
...
+			]
+		},
+		{
+			"#": "36",
+			"label": "Default Dim Value when turned ON via paddle press",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x000c/hs-wx300.json]:
```suggestion
			"label": "Default Brightness (Manual Control)",
```

File: packages/config/config/devices/0x000c/hs-wx300.json [hunk 5]
```diff
@@ -161,6 +161,81 @@
...
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 99,
+			"defaultValue": 0,
+			"unsigned": true
```
Comment (AlCalzone) [inline packages/config/config/devices/0x000c/hs-wx300.json]:
```suggestion
			"options": [
				{
					"label": "Last non-zero brightness",
					"value": 0
				}
			]
```
unsigned isn't necessary unless the check complains :)

File: packages/config/config/devices/0x000c/hs-wx300.json [hunk 6]
```diff
@@ -161,6 +161,81 @@
...
+			"options": [
+				{
+					"label": "Load is controlled with paddle",
+					"value": 0
+				},
+				{
+					"label": "Load is not controlled with paddle",
+					"value": 1
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x000c/hs-wx300.json]:
```suggestion

			"$import": "~/templates/master_template.json#base_enable_disable_inverted",
			"label": "Control Load With Paddle",
			"defaultValue": 0
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
If the parameters were added in v1.13, they should be made [conditional](https://zwave-js.github.io/node-zwave-js/#/config-files/conditional-settings?id=conditional-configuration-settings).

Also, the labels don't quite match our requirements.

**Comment by AlCalzone** [comment]
Sorry for taking so long to review, the v10 release had me quite busy.

**Comment by AlCalzone** [comment]
Thanks, that looks better. Heads up: I'm going to test a few things on your PR - seems that I recently broke  a check.

Just one tip: If you plan to contribute in the future, it's better to work in a branch, not master. Less headaches when your repo is out of date. Also, not force pushing makes it easier to follow the changes during review.

**Comment by McLain114** [comment]
@AlCalzone apologies. I'll keep that in mind.

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**